source/jormungandr/tests/direct_path_asgard_integration_tests.py

```diff
@@ -149,6 +170,7 @@ def test_journey_with_bike_direct_path(self):
         assert response['journeys'][1]['durations']['total'] == 1000
         assert response['journeys'][1]['distances']['bike'] == 100
         assert response['journeys'][1]['duration'] == 1000
+        assert response['journeys'][1]['sections'][0]['cycle_lane_length'] == 30.0
```

Comment:
It is cast as 'int' in 'add_cycle_lane_length', isn't it?

Reply:
minor (for readability): So shouldn't you have :
```suggestion
        assert response['journeys'][1]['sections'][0]['cycle_lane_length'] == 30
```